tests/unit/dataset/dataset_library_test.py

```diff
@@ -19,45 +19,49 @@
 # SOFTWARE.
 """Test dataset library."""
 import glob
+from dataclasses import dataclass
 from pathlib import Path
 from unittest import mock
 
 import pytest
 import yaml
 
 import pymovements as pm
```

Comment:
we still need this, right?

Reply:
yes, it's needed for the new `test_register_definition_class()` (although maybe it could also work without in this test?)

Reply:
I just removed the import let's see if it breaks

Reply:
seems to work without, I resolve this conversation myself for auto-merge